src/settings.rs

```diff
@@ -30,7 +38,7 @@ pub const R_VSYNC: console::CVar<bool> = console::CVar {
     default: &|| false,
 };
 
-pub const CL_MASTER_VOLUME: console::CVar<i64> = console::CVar {
+pub const CL_MASTER_VOLUME: console::CVar<u8> = console::CVar {
```

Comment:
Are you sure, that u8 is okay here as well?

Reply:
Volume is a value between 0 and 100, how is this not okay?

Reply:
Just wanted to make sure, that it is okay.